frontend/src/hooks/api/auditLogs/types.tsx

```diff
@@ -679,6 +679,34 @@ interface OrgAdminAccessProjectEvent {
   }; // no metadata yet
 }
 
+interface CreateCertificateTemplate {
+  type: EventType.CREATE_CERTIFICATE_TEMPLATE;
+  metadata: {
```

Comment:
Add fields used when creating certificate template to the audit log metadata (mentioned elsewhere) for more complete audit trail.